### ISSUE TITLE
无法在cmd中粘贴token/ Cannot paste token in cmd Body:

### ISSUE THREAD

**Issue by shilonglongya** [huggingface/diffusers#2735]
I am trying to convert the safetensors model to an onnx compatible model, following this tutorial (https://huggingface.co/transformers/serialization.html). When I log in to hugging face in cmd, I need to enter a token, but I cannot paste my token in the token text box. I have copied my token, but I cannot paste it into the text box using either CTRL+V or the right mouse button. This results in an Invalid token passed error. Is there any way to solve this problem? Thank you!
我正在尝试将safetensors模型转换成onnx可用的模型，我按照这个教程（https://huggingface.co/transformers/serialization.html）进行操作，在cmd中登录hugging face时需要输入token，但是我无法在token文本框中粘贴我的token。我已经复制了我的token，但是无论我使用CTRL+V还是鼠标右键都不能将其粘贴到文本框中。这导致了Invalid token passed的错误。请问有什么办法可以解决这个问题吗？谢谢！

**Comment by sayakpaul** [comment]
This is something I couldn't reproduce on my end. Could you provide any screenshot if you have one?

**Comment by shilonglongya** [comment]
> 这是我无法重现的东西。如果您有任何屏幕截图，您能提供任何屏幕截图吗？
I copied my token from the website, but when I need to enter the token, whether I use Ctrl+v or right-click, whether or not I use administrator privileges, I cannot paste my token. At the same time, I saw people on the Internet say that the token will not be displayed in the terminal when pasted. I also tried to run the program after right-clicking or Ctrl+V and pressing Enter, but the program still reported an error because I did not enter a token. Here is my screenshot.
![QQ图片20230322212820](https://user-images.githubusercontent.com/96930391/226920854-722742bb-12a6-408e-834f-b8f44bcd0793.png)
